app/views/admin/loans/content/_table_question.html.slim

```diff
@@ -0,0 +1,89 @@
+.table-responsive.editable-tables
+
+  / TODO: Convert input to expected input for loan question
+  input type="hidden" data-container="breakeven"
+
+  .table-container
+    h4.title = t("loan.breakeven.products", count: 2)
+
+    table.table.table-condensed.table-striped.table-bordered.editable-table data-table="products"
+      thead
+        tr
+          th = t("loan.breakeven.product_name")
+          th.wide = t("loan.breakeven.description")
+          th = t("loan.breakeven.unit")
+          th.narrow = t("loan.breakeven.price")
+          th.narrow = t("loan.breakeven.cost")
+          th.narrow = t("loan.breakeven.quantity")
+          th.narrow: .actions: a data-action="add": i.fa.fa-plus-circle
+      tbody
+        / Hidden row is used to generate new rows
+        / Is at top instead of bottom to not affect striped rows
+        tr.ui-state-default.hidden data-group="product"
```

Comment:
Ideally I would not be repeating myself and these would be separate partials. But then we might end up with lots of tiny partials once there are more tables. I like tiny partials, but know that isn't preferred in this project.

However, there are 3 groups of rows. One is hidden as described above. One is for actual data that is being loaded into the form. And one is an empty row to start out with whether or not the table has any existing data.